pod/perlstyle.pod

```diff
@@ -23,7 +23,7 @@ well as several other useful L<named features|feature/AVAILABLE FEATURES>).
 
     use v5.36;
 
-Regarding aesthetics of code lay out, about the only thing Larry
+Regarding aesthetics of code layout, the only thing Larry
```

Comment:
including 'about' is more correct than removing it, as its use indicates there may be other things not listed here, which gives wiggle room to us if something comes up

Reply:
Double "about" sounds weird

Reply:
@khwilliamson ok I added "about" back, now it's only "lay out" that's wrong.

Could you squash and merge via GitHub or should I squash the three commits myself?

Reply:
I can do the squash, but it's a pain for me.  It's should be easier for you.  If you don't know how to do it, it would be good to learn for future use.  Someone can guide you through.  email me to get started.

Reply:
Done (using GitHub Desktop). I can do via CLI too, but it's just easier.

Sorry, why it's a pain for you, because of GitHub UI?

There's no such a button in GitHub?:

https://docs.gitlab.com/ee/user/project/merge_requests/squash_and_merge.html

Thanks